gem_city.rb

```diff
@@ -0,0 +1,62 @@
+# This class represents the town of GemCity
+# This is a town riddled with crime but we can find out how happy the town is
+class GemCity
+  attr_reader :population
+  def initialize
+    @people = {
+      thieves:  5,
+      Officers: 1
```

Comment:
Actually, what about ditching the hash for two instance variables (`@thieves` & `@officers`)?